Server/VikingVault.Services/CompanyService.cs

```diff
@@ -73,9 +73,9 @@ public List<CompanyDataDTO> GetAllCompanies()
                 var companies = _dbContext.User.Where(user => user.Role.Type == RoleEnum.Company.ToString()).ToList();
                 foreach (var company in companies)
                 {
-                    var account = _dbContext.BankAccount
+                    var account = _dbContext.BankAccount.Include(bankAccount => bankAccount.User)
                         .SingleOrDefault(bankAccount => bankAccount.User.Id == company.Id &&
-                                                        bankAccount.CurrencyType == CurrencyEnum.Ron.ToString());
+                                                        bankAccount.CurrencyType.Equals("Ron"));
```

Comment:
why did you change this?

Reply:
I think it is safer to use equals for strings, instead of == operator.